example/dj/apps/issue_tracker/cores/resources.py

```diff
@@ -12,3 +12,10 @@ def get(self):
             'created': user.created_issues.count(),
             'watching': user.watching_issues.count(),
         }
+
+
+class UserModelResource(RESTModelResource):
+
+    def watching_issues_count(self, obj):
+        return obj.watching_issues.count()
+    watching_issues_count.short_description = 'watching count'
```

Comment:
Why are you not using decorator?

Reply:
because I want test this solution. Sometimes in some library cannot be imported is_core library (because it is not requirement) and is used this solution. So I want test it